yarn-project/simulator/src/client/test_utils.ts

```diff
@@ -0,0 +1,35 @@
+import { Fq, Fr, GeneratorIndex, Point } from '@aztec/circuits.js';
+import { Grumpkin } from '@aztec/circuits.js/barretenberg';
+import { pedersenHash } from '@aztec/foundation/crypto';
+
+// Copied over from `noir-projects/aztec-nr/aztec/src/generators.nr`
+const G_SLOT = new Point(
+  new Fr(0x041223147b680850dc82e8a55a952d4df20256fe0593d949a9541ca00f0abf15n),
+  new Fr(0x0a8c72e60d0e60f5d804549d48f3044d06140b98ed717a9b532af630c1530791n),
+  false,
+);
+
+/**
+ * Computes a note hiding point as is done by the default implementation injected by macros.
+ * @param noteContent - The note content (e.g. note.items).
+ * @returns A note hiding point.
+ */
+export function computeNoteHidingPoint(noteContent: Fr[]): Point {
+  // TODO(#7551): how this is computed will need to be updated
```

Comment:
Why?

Reply:
Because the way how I compute it now is hacky and I plan on replacing it with Pedersen commitment once it's ready.